src/index.ts

```diff
@@ -127,7 +127,7 @@ const childProcessArgs = () => {
 export const esbuildDev = async (options: Options) => {
   const workspaceRoot = findWorkspaceRoot(process.cwd()) || process.cwd();
   const workDir = await fs.mkdtemp(path.join(os.tmpdir(), "esbuild-dev"));
-  const syncSocketPath = path.join(workDir, "ipc.sock");
+  const syncSocketPath = path.join("\\\\?\\pipe", workDir, "ipc.sock");
```

Comment:
My fault for not having tests yet, but does this still work on *nixes? I've never seen this syntax before!

Reply:
Oh that's my fault, I will test & fix it on *nix (in fact MacOS) now.